src/main/java/eu/europa/ted/eforms/sdk/domain/field/CodeListPropertyValue.java

```diff
@@ -16,10 +15,7 @@ public class CodeListPropertyValue implements Serializable {
    */
   private String id;
 
-  /**
-   * Use the generated type enum.
-   */
-  private CodeListType type;
+  private String type;
```

Comment:
This removes the implicit check that "type" has one of the expected values.
I think it would be better to have the corresponding enums in this project.
It will require manual updates if the enum is changed, but it is safer.

The same applies for the other enums removed in this commit.

Reply:
But do we actually need this kind of check in this tool?
The enums were removed based on a related discussion we had some time ago.
Both to get rid of the mdd-entities dependency and to not have to update the tool after almost every change on the DB.

Reply:
Fixed.
The used enum classes have now been copied from MDD and mdd-entities.